charts/vela-core/templates/defwithtemplate/expose.yaml

```diff
@@ -0,0 +1,36 @@
+# Code generated by KubeVela templates. DO NOT EDIT.
+apiVersion: core.oam.dev/v1beta1
+kind: TraitDefinition
+metadata:
+  annotations:
+    definition.oam.dev/description: "Expose port to enable web traffic for your component."
```

Comment:
This will be really confusing with web service component, not sure how we will expect user use this. To expose port in a chart? I believe most chart will package service as part of the component.